src/dataflow/components/dataflow-program-cover.sass

```diff
@@ -8,7 +8,7 @@
   display: flex
   align-items: center
   justify-content: center
-  top: 40px
+  top: 0px
```

Comment:
One side effect of this is that the program cover is over the topbar when the program is running.